core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-announcements/store/dot-announcements.store.ts

```diff
@@ -70,6 +80,45 @@ export class AnnouncementsStore extends ComponentStore<DotAnnouncementsState> {
         (state) => state.showUnreadAnnouncement
     );
 
+    readonly selectKnowledgeCenterLinks: Signal<AnnouncementLink[]> = this.selectSignal((state) => {
+        return [
+            {
+                id: '1',
+                label: this.dotMessageService.get('announcements.knowledge.center.documentation'),
+                url: `https://www.dotcms.com/docs/latest/table-of-contents?${state.utmParameters}`
+            },
+            {
+                id: '2',
+                label: this.dotMessageService.get('announcements.knowledge.center.blog'),
+                url: `https://www.dotcms.com/blog/?${state.utmParameters}`
+            },
+            {
+                id: '3',
+                label: this.dotMessageService.get('announcements.knowledge.center.forum'),
+                url: 'https://groups.google.com/g/dotcms'
+            }
+        ];
+    });
+
+    readonly selectContactLinks: Signal<AnnouncementLink[]> = this.selectSignal((state) => {
+        return [
+            {
+                id: '1',
+                label: this.dotMessageService.get('announcements.contact.customer.support'),
+                url: `https://www.dotcms.com/services/support/?${state.utmParameters}`
+            },
+            {
+                id: '2',
+                label: this.dotMessageService.get('announcements.contact.professional.services'),
+                url: `https://www.dotcms.com/services/professional-services/?${state.utmParameters}`
+            }
+        ];
+    });
```

Comment:
SQ can see this as duplicated code, you can change the order of the properties in the object to make it pass the tests